src/Elements/Core/Icon/Icon.vue

```diff
@@ -18,7 +18,7 @@
 import IconCheckSmallLine from '@/assets/images/icon/icon-check-small-line.svg?inline';
 import IconPlusSmallLine from '@/assets/images/icon/icon-plus-small-line.svg?inline';
 import IconArrowSmallLine from '@/assets/images/icon/icon-arrow-small-line.svg?inline';
-import IconOverflowSmallFill from '@/assets/images/icon/icon-overflow-small-fill.svg?inline';
+import IconThreedotSmallFill from '@/assets/images/icon/icon-threedot-small-fill.svg?inline';
```

Comment:
```suggestion
import IconThreeDotSmallFill from '@/assets/images/icon/icon-threeDot-small-fill.svg?inline';
```
이렇게 하면 어떨까요?
바로 아래 TextArrow케이스를 따르면 좋을 것 같아요

Reply:
그게 더 좋겠네요